File: Python/Easy/344. Reverse String.py
# https://leetcode.com/problems/reverse-string/description/

class Solution:
    def reverseString(self, s: List[str]) -> None:
        n = len(s) - 1
        i = 0
        while i <= n - i:
            s[i], s[n - i] = s[n - i], s[i]
            i += 1

File: Java/Easy/344. Reverse String.java
// https://leetcode.com/problems/reverse-string/description/

class Solution {
    public void reverseString(char[] s) {
        int n = s.length;
        for(int i = 0; i < n/2; i++) {
            char tmp = s[i];
            s[i] = s[n - i - 1];
            s[n - i - 1] = tmp;
        }
    }
}